frontend/src/components/ResultPage.tsx

```diff
@@ -0,0 +1,24 @@
+interface ResultProps {
+  result?: string
```

Comment:
Given this data comes from backend it's most likely some kind of JSON object and not a plain string. It's the best to check if there's an API contract (swagger doc) of how the data would look like.